grafana-plugin/src/module.ts

```diff
@@ -1,15 +1,48 @@
 import { ComponentClass } from 'react';
 
-import { AppPlugin } from '@grafana/data';
+import { AppPlugin, PluginExtensionPoints } from '@grafana/data';
 
+import { MobileAppConnectionWrapper } from 'containers/MobileAppConnection/MobileAppConnection';
 import PluginConfigPage from 'containers/PluginConfigPage/PluginConfigPage';
 import { GrafanaPluginRootPage } from 'plugin/GrafanaPluginRootPage';
+import { getGrafanaVersion } from 'plugin/GrafanaPluginRootPage.helpers';
+import { IRM_TAB } from 'utils/consts';
 
 import { OnCallPluginConfigPageProps, OnCallPluginMetaJSONData } from './types';
 
-export const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
+const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
   title: 'Configuration',
   icon: 'cog',
   body: PluginConfigPage as unknown as ComponentClass<OnCallPluginConfigPageProps, unknown>,
   id: 'configuration',
 });
+
+if (isUseProfileExtensionPointEnabled()) {
+  // TODO Change this to use the PluginExtensionPoints enum once that is available in @grafana/data
+  const extensionPointId = 'grafana/user/profile/tab';
+
+  plugin.configureExtensionComponent({
+    title: IRM_TAB,
+    description: 'IRM settings',
+    extensionPointId,
+    /**
+     * typing MobileAppConnectionWrapper as any until 10.2.0 is released
+     * https://github.com/grafana/grafana/pull/75019#issuecomment-1724997540
+     */
```

Comment:
I believe I copied this comment from another repo where I found how they were doing this.. this comment might be irrelevant. You could try removing the `as any` and see if ts complains or not?

Reply:
Right, not needed.